core/controllers/feedback_test.py

```diff
@@ -602,49 +605,50 @@ def setUp(self):
         csrf_token = self.get_csrf_token_from_response(response)
 
         # Create suggestions.
-        self.post_json(
-            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
-                'exploration_version': 3,
-                'state_name': u'State A',
-                'description': u'Suggestion for state A.',
-                'suggestion_html': 'new accepted suggestion for state A',
-            }, csrf_token=csrf_token)
-        self.post_json(
-            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
-                'exploration_version': 1,
-                'state_name': u'State 2',
-                'description': u'A new value.',
-                'suggestion_html': 'some new value',
-            }, csrf_token=csrf_token)
-        self.post_json(
-            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
-                'exploration_version': 2,
-                'state_name': u'State 3',
-                'description': u'Empty suggestion',
-                'suggestion_html': '',
-            }, csrf_token=csrf_token)
-        self.post_json(
-            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
-                'exploration_version': 2,
-                'state_name': u'State A',
-                'description': u'Just a space.',
-                'suggestion_html': ' ',
-            }, csrf_token=csrf_token)
-        self.post_json(
-            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
-                'exploration_version': 1,
-                'state_name': u'State 2',
-                'description': u'Random characters.',
-                'suggestion_html': '#!$%',
-            }, csrf_token=csrf_token)
-        self.post_json(
-            '%s/%s' % (feconf.SUGGESTION_URL_PREFIX, self.EXP_ID), {
-                'exploration_version': 2,
-                'state_name': u'State 3',
-                'description': u'Very bizarre characters.',
-                'suggestion_html': u'Ֆݓॵক',
-            }, csrf_token=csrf_token)
-        self.logout()
+        with self.swap(constants, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', False):
```

Comment:
Better to put all the "False" tests in a feedback_old_test.py file, so it can be deleted after the flag is flipped for good.

Reply:
Moved the old feedback tests to a different suite, also made a suite to test the new suggestions framework with the old feedback framework.